src/VisualStudio/Core/Test.Next/Remote/RemoteHostClientServiceFactoryTests.cs

```diff
@@ -124,9 +125,9 @@ public async Task UpdaterService()
             var listenerProvider = exportProvider.GetExportedValue<AsynchronousOperationListenerProvider>();
             var analyzerReference = new AnalyzerFileReference(typeof(object).Assembly.Location, new NullAssemblyAnalyzerLoader());
 
-            var service = CreateRemoteHostClientService(workspace, SpecializedCollections.SingletonEnumerable<AnalyzerReference>(analyzerReference), listenerProvider);
+            var (service, experimentationService) = await CreateRemoteHostClientServiceAsync(workspace, SpecializedCollections.SingletonEnumerable<AnalyzerReference>(analyzerReference), listenerProvider);
 
-            service.Enable();
+            service.Enable(experimentationService);
```

Comment:
Does every test do `service.Enable(experimentationService)`? Should we just unify?

Reply:
➡️ I'm going to leave this for a follow-up.